app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,25 @@
   </head>
 
   <body>
-    <%= yield %>
+    <nav>
+      <div class="nav-wrapper">
+        <a href="/" class="brand-logo">Foodie Moodie</a>
+        <ul id="nav-mobile" class="right">
```

Comment:
I see that you add css classes but there is no css framework added to the project.